src/dss/LerpAbstract.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity >=0.5.12;
+
+// https://github.com/makerdao/dss-lerp/blob/master/src/Lerp.sol
+interface LerpAbstract {
+    function target() external view returns (address);
+    function what() external view returns (bytes32);
+    function start() external view returns (uint256);
+    function end() external view returns (uint256);
+    function duration() external view returns (uint256);
+    function done() external view returns (bool);
+    function startTime() external view returns (uint256);
+    function tick() external;
```

Comment:
```suggestion
    function tick() external;
    function ilk() external view returns (bytes32);
```

Reply:
This only exists for the ilk lerp, but I suppose it's fine to include it.